### ISSUE TITLE
1856: Warn if corporation doesn't have enough treasury to cover interest

### ISSUE THREAD

**Issue by ventusignis** [tobymao/18xx#5569]
https://18xx.games/game/44516?action=253
In this game at this action the WR has $40 interest due but only $15 in the treasury; the interest due should be highlighted yellow to indicate this will have consequences (penalties to the route revenue)

**Comment by jenf** [comment]
didn't this get fixed?

**Comment by ventusignis** [comment]
No, this is separate from warning if a corporation has more than max mloans
